README.md

```diff
@@ -234,30 +234,31 @@ This ensures that we split only by those chromosomes for which we have records
              --vep-path ~{vepPath} --vep-data ~{vepCacheDir} \
              --min-hom-vaf ~{minHomVaf} --buffer-size ~{bufferSize} \
              $retainInfo_command_line \
-             --vep-stats ~{vepStats}
-```
+             --vep-stats ~{vepStats} \
+             --custom-enst ~{customTranscriptENSTids}
+ ```
  
-### mergeMafs
+ ### mergeMafs
```

Comment:
Check for these leading extra spaces, they will prevent proper rendering of an .md file (you should check that README.md looks Ok on your branch before merging)